Weather App/view/ForecastWeatherTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ForecastWeatherTableViewCell.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 17/03/22.
+//
+
+import UIKit
+
+public class ForecastWeatherTableViewCell: UITableViewCell {
+    
+    @IBOutlet public weak var DayLabel: UILabel!
+    @IBOutlet public weak var DayConditionImage: UIImageView!
+    @IBOutlet public weak var MinTempLabel: UILabel!
+    @IBOutlet public weak var AvgTempLabel: UILabel!
+    @IBOutlet public weak var MaxTempLabel: UILabel!
+    
+    public static let id : String = "forecastWeatherTableCellID"
```

Comment:
You can introduce a generic property for these static ids.

Reply:
please attach resources for this, couldn't find anything around this.

Reply:
Something like this https://cocoacasts.com/dequeueing-reusable-views-with-generics-and-protocols